apps/admin-gui/src/app/shared/components/services-list/services-list.component.html

```diff
@@ -34,8 +46,21 @@
           <td *matCellDef="let service" mat-cell>{{service.name}}</td>
         </ng-container>
         <ng-container matColumnDef="enabled">
-          <th *matHeaderCellDef mat-header-cell>{{'SHARED.COMPONENTS.SERVICES_LIST.ENABLED' | translate}}</th>
-          <td *matCellDef="let service" mat-cell>{{service.enabled}}</td>
+          <th *matHeaderCellDef mat-header-cell></th>
+          <td *matCellDef="let service" mat-cell>
+            <mat-icon
+            class="align-bottom"
+            *ngIf="service.enabled"
+            matTooltip="{{'SHARED.COMPONENTS.SERVICES_LIST.ENABLED' | translate}}">
+            lock_open
+          </mat-icon>
+            <mat-icon
+              class="align-bottom"
+              *ngIf="!service.enabled"
+              matTooltip="{{'SHARED.COMPONENTS.SERVICES_LIST.DISABLED' | translate}}">
+              lock
+            </mat-icon>
```

Comment:
So I have been consulting it with some other people and we have choose this icons:
enabled - https://material.io/resources/icons/?icon=check_circle_outline&style=baseline
disabled - https://material.io/resources/icons/?icon=block&style=baseline